ovos_backend_client/backends/neon.py

```diff
@@ -0,0 +1,202 @@
+        """Issue an API call and map the return value into a weather report
+
+        Args:
+            units (str): metric or imperial measurement units
+            lat_lon (tuple): the geologic (latitude, longitude) of the weather location
+        """
+        raise NotImplementedError()
+
+    def owm_get_daily(self, lat_lon=None, lang="en-us", units="metric"):
+        """Issue an API call and map the return value into a weather report
+
+        Args:
+            units (str): metric or imperial measurement units
+            lat_lon (tuple): the geologic (latitude, longitude) of the weather location
+        """
+        raise NotImplementedError()
+
+    # Wolfram Alpha API
+    def wolfram_spoken(self, query, units="metric", lat_lon=None, optional_params=None):
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        lat, lon = lat_lon
+        return self._request_mq_api(
+            "wolfram_alpha",
+            {
+                "units": units,
+                "lat": lat,
+                "lon": lon,
+                "query": query,
+                "api": "spoken"
+            })
+
+    def wolfram_simple(self, query, units="metric", lat_lon=None, optional_params=None):
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        lat, lon = lat_lon
+        return self._request_mq_api(
+            "wolfram_alpha",
+            {
+                "units": units,
+                "lat": lat,
+                "lon": lon,
+                "query": query,
+                "api": "simple"
+            })
+
+    def wolfram_full_results(self, query, units="metric", lat_lon=None, optional_params=None):
+        """Wrapper for the WolframAlpha Full Results v2 API.
+        https://products.wolframalpha.com/api/documentation/
+        Pods of interest
+        - Input interpretation - Wolfram's determination of what is being asked about.
+        - Name - primary name of
+        """
+        optional_params = optional_params or {}
+        if not lat_lon:
+            lat_lon = self._get_lat_lon(**optional_params)
+        lat, lon = lat_lon
+        return self._request_mq_api(
+            "wolfram_alpha",
+            {
+                "units": units,
+                "lat": lat,
+                "lon": lon,
+                "query": query,
+                "api": "full"
+            })
+
+    # Geolocation Api
+    def geolocation_get(self, location):
+        """Call the geolocation endpoint.
+
+        Args:
+            location (str): the location to lookup (e.g. Kansas City Missouri)
+
+        Returns:
+            str: JSON structure with lookup results
+        """
+        raise NotImplementedError()  # TODO
+
+    # Metrics API
+    def metrics_upload(self, name, data):
+        """ upload metrics"""
+        raise NotImplementedError()  # TODO
```

Comment:
@NeonDaniel could you please implement these missing methods since you are the most familiar with the mq messages?